code42.json

```diff
@@ -1746,6 +1746,171 @@
                 "type": "table"
             },
             "versions": "EQ(*)"
+        },
+        {
+            "action": "add legalhold user",
+            "identifier": "add_legalhold_user",
+            "description": "Add a user (custodian) to a legal hold matter",
+            "verbose": "",
+            "type": "contain",
```

Comment:
It probably makes sense to make use of the `undo` functionality here like we do with the detection lists.